### ISSUE TITLE
gedit: no color named 'white'

### ISSUE THREAD

**Issue by Paz-it** [ubuntu/yaru#667]
gtk.css line 6158:
.gedit-document-panel row:hover:selected button:hover {
  color: white; } <<<<< 
gtk-dark.css line 6148:
.gedit-document-panel row:hover:selected button:hover {
  color: white; }<<<<<

**Comment by clobrano** [comment]
@Paz-it  how did you get this log? Running gedit from command line does not print any warning message

**Comment by clobrano** [comment]
@Paz-it could you update on this?

**Comment by Paz-it** [comment]
@clobrano, I saw it repeating many times in  "logs"

**Comment by clobrano** [comment]
I am not able to reproduce it

![image](https://user-images.githubusercontent.com/2883614/43582474-dba82c9a-965c-11e8-8c86-f7a5af211b5c.png)

**Comment by clobrano** [comment]
I was on 18.04, `git describe` is just to say that I am using master branch up to date

> maybe you can just try the hex numbers and see if it will make it go away? It's SPAMMING the logs and annoying.

The problem is that I cannot reproduce the problem. I see no message in logs, so I cannot test the solution.

**Comment by clobrano** [comment]
@madsrh , @Feichtmeier, can you reproduce this?

**Comment by madsrh** [comment]
@clobrano Nope

**Comment by Paz-it** [comment]
Any chance there is a connection to that issue?  https://github.com/ubuntu/yaru/issues/686
@clobrano @madsrh

**Comment by clobrano** [comment]
Tried that as well. When clicking on icons in sidebar I don't see any warning message

**Comment by Paz-it** [comment]
@clobrano, At start!
 I think I'm "on" something: 
In the code I added there is Background element for selection unlike the other elements: 
.gedit-document-panel row:hover button:hover {
  background-color: rgba(61, 61, 61, 0.15); }<<<<<<

.gedit-document-panel row:hover button:active {
  background-color: rgba(61, 61, 61, 0.25); }<<<<<<<<<

.gedit-document-panel row:hover:selected button:hover {
  color: white; }
The warnings are as you can see in my terminal above.
so maybe since the background is missing and can't be parsed the white color don't meaning.
Just a logical guess.

**Comment by clobrano** [comment]
> so maybe since the background is missing

@Paz-it consider that when a property does not appear in some style it does not mean that it is "missing", but that that property keeps the same value defined elsewhere (e.g. the one defined in normal state)

> the white color doesn't have a reference.

background-color and color are two independent properties, I don't understand why you say that

I had a look at the actual yaru's code:

   row:hover:selected button:hover {
        color: $selected_fg_color;
    }

`$selected_bg_color` is actually `white`, this means that white is used in many places without any problem.

Finally, the element `.gedit-document-panel row:hover:selected button:hover` is used to style the hover effect of close button in the list of opened documents in gedit sidebar. I tried many changes and nothing showed the problem you reported

I could suggest you to retry replacing "white" with "#FFFFFF" just to try everything, but

**Comment by Paz-it** [comment]
@clobrano , It was just a guess. I know  that basics. 
Color: refers to the text,  etc...  
I was just looking for the reason for the strange warning (that appears only on my side) :smile: 

Another place I think I need to look is: 
"GtkSourceView"  Which is acting out strangely in my side. Since my OS is not fresh install it might be something of a leftover from previous OS, Ubuntu 17.10 and also have in mind it's a 32 bit not 64.

Bottom line : It might be a local problem on my side only.
Thanks for looking at it.

**Comment by clobrano** [comment]
> Bottom line : It might be a local problem on my side only.

since it looks so to me as well, I would close this ticket. If you come up with a different result however, feel free to reopen it.
